packages/components/src/test/common/ContentBuilder.test.ts

```diff
@@ -127,4 +127,11 @@ describe("PropertyRecordsBuilder", () => {
     });
   });
 
+  it("Sets editor name when field info types typeName is Number", () => {
```

Comment:
the convention is to start test names in lowercase
```suggestion
  it("sets editor name when field info types typeName is Number", () => {
```